src/crypto/crypto_tls.cc

```diff
@@ -144,32 +150,47 @@ int NewSessionCallback(SSL* s, SSL_SESSION* sess) {
     return 0;
 
   // Serialize session
-  // TODO(@jasnell): An AllocatedBuffer or BackingStore would be better
```

Comment:
I'm actually not very sure about what improvement this provides as I have just inlined the code here; would be nice to hear an explanation for this TODO